### ISSUE TITLE
Latest icons to add

### ISSUE THREAD

**Issue by xzj8b3** [bikass/korla#34]
Please add the following icons to make the theme really beautiful:

Conky (too small a size compared to the others)

Cheese (If you can a more pleasant icon)

Dvd95  

OpenJDK

**Comment by bikass** [comment]
Hi, all icons you mentioned are in theme. I added one symlink for Conky, maybe now icon will show up.
Cheese icon has been changed. You can see it in latest commits (it's not in releases yet).

**Comment by xzj8b3** [comment]
> Hi, all icons you mentioned are in theme. I added one symlink for Conky, maybe now icon will show up.
> Cheese icon has been changed. You can see it in latest commits (it's not in releases yet).

DVD95 still has a standard icon (or should it be set manually?)

OpenJDK is standard and it's really out of tune with others!!!!

**Comment by bikass** [comment]
Are you seeing these icons?
https://github.com/bikass/korla/blob/master/korla/apps/scalable/dvd95.svg
https://github.com/bikass/korla/blob/master/korla/apps/scalable/java.svg

OpenJDK is symlink to java. It is the same in other themes.

**Comment by xzj8b3** [comment]
> Stai vedendo queste icone? 
> [https://github.com/bikass/korla/blob/master/korla/apps/scalable/dvd95.svg ](https://github.com/bikass/korla/blob/master/korla/apps/scalable/dvd95.svg)
> https://github.com/bikass/korla/blob/master/korla/apps/scalable/java.svg
> 
> OpenJDK è un link simbolico a java. È lo stesso in altri temi.

It's perfect because the one in Dvd95 is a bit old like the one in OpenJDK 1.8.0

So they're already there and need to be set manually??

**Comment by bikass** [comment]
Hi, I don't really understand issue. You don't like icons in theme or you see other icons?

**Comment by xzj8b3** [comment]
> Ciao, non capisco davvero il problema. Non ti piacciono le icone in tema o vedi altre icone?

In Fedora 30 I see defaulth icons of those software

**Comment by bikass** [comment]
Added symlinks to openjdk. It works now.
DVD95 still can't find solution

**Comment by xzj8b3** [comment]
> Aggiunto collegamenti simbolici a openjdk. Ora funziona. 
> DVD95 non riesce ancora a trovare una soluzione

Well it is not so urgent Dvd95....so much so if I notice others I will point them out because it is the most lively theme

**Comment by xzj8b3** [comment]
> Aggiunto collegamenti simbolici a openjdk. Ora funziona. 
> DVD95 non riesce ancora a trovare una soluzione

1.1.4 really nice we expect improvements as Dvd95 ,Devede-NG ,OnionShare,TimeShift 

Too pretty for you to leave those of software defaulth

**Comment by bikass** [comment]
I have no idea whats wrong with those icons. All of them are in the theme. I checked other themes, they have same icons...

**Comment by xzj8b3** [comment]
> Non ho idea di cosa non vada con quelle icone. Tutti loro sono nel tema. Ho controllato altri temi, hanno le stesse icone ...
Yes, but those mentioned have icons of defaulth, you draw them much more beautiful

**Comment by xzj8b3** [comment]
> Non ho idea di cosa non vada con quelle icone. Tutti loro sono nel tema. Ho controllato altri temi, hanno le stesse icone ...

Yes, but those mentioned have icons of defaulth, you draw them much more beautiful
